ocaml/xapi/xapi_host_helpers.ml

```diff
@@ -411,8 +411,7 @@ let consider_enabling_host_nolock ~__context =
       debug
         "Host.enabled: system has just restarted: remove livepatch failure \
          guidances" ;
-      Db.Host.remove_pending_guidances ~__context ~self:localhost
-        ~value:`reboot_host ;
+      remove_pending_guidance ~__context ~self:localhost ~value:`reboot_host ;
```

Comment:
Should we use the same helper function for the three livepatch-failure cases below?

Reply:
I think as in our design, livepatch-failure guidance will be added only to the recommended guidance list.

Reply:
Clearing 4 `reboot_host* tasks from 3 lists could be wrapped into one helpful. It would be safer in case there are any bugs.